error_handler/build.gradle

```diff
@@ -0,0 +1,116 @@
+buildscript {
+    ext {
+        junitVersion = '5.3.2'
+        springFoxVersion = '2.9.2'
+        jacksonVersion = '2.9.8'
+    }
+}
+
+plugins {
+    id 'io.franzbecker.gradle-lombok' version '1.14'
+    id "org.owasp.dependencycheck" version "4.0.0.1"
+    id "com.github.spotbugs" version "1.6.5"
+    id 'pmd'
+    id 'checkstyle'
+}
+
+apply plugin: 'java'
+
+lombok {
+    version = '1.18.4'
+    sha256 = ""
+}
+
+group = 'uk.gov.dhsc.htbhf'
+sourceCompatibility = 1.11
+targetCompatibility = 1.11
+
+repositories {
+    mavenCentral()
+    maven {
+        url  "https://dl.bintray.com/departmentofhealth-htbhf/maven"
+    }
+}
+
+
+dependencies {
+    implementation "org.springframework:spring-web:${springVersion}"
+    implementation "org.springframework.boot:spring-boot-autoconfigure:${springBootVersion}"
+    implementation "org.springframework:spring-webmvc:${springVersion}"
+    implementation "org.springframework:spring-context:${springVersion}"
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonVersion}"
+    implementation "com.google.code.findbugs:annotations:3.0.1"
+    implementation "org.slf4j:slf4j-api:1.7.25"
+    implementation "org.apache.tomcat.embed:tomcat-embed-core:9.0.12"
+    implementation "io.springfox:springfox-swagger2:${springFoxVersion}"
+    implementation "io.springfox:springfox-bean-validators:${springFoxVersion}"
+    implementation "uk.gov.dhsc.htbhf:htbhf-request-context:0.0.2"
```

Comment:
Can we make this inherit from the project version? (Not sure if we can, might be worth a try)

Reply:
We can't as that version won't exist in bintray until after the build.

Reply:
Can't we have a reference like in Maven to `${project.version}`, Gradle should have an equivalent. If not, we should be using Maven. :-)

Reply:
It does but the build will fail as the request-context version won't exist anywhere until after the build.